.github/workflows/build.yml

```diff
@@ -98,4 +100,4 @@ jobs:
         env:
           GH_TOKEN: ${{ secrets.GH_TOKEN }}
           PYPI_TOKEN: ${{ secrets.PYPI_TOKEN }}
-        run: npx semantic-release  
+        run: npx semantic-release  
```

Comment:
don't remove this

Reply:
oh I did not, I just removed a trailing blank line

Reply:
Yeah, I just mean don't remove the trailing blank line